TheWarWithin/Priorities/HunterMarksmanship.simc

```diff
@@ -23,8 +21,8 @@ actions+=/call_action_list,name=drtrickshots,strict=1,if=active_enemies>2&talent
 actions+=/call_action_list,name=senttrickshots,strict=1,if=active_enemies>2&talent.trick_shots&!hero_tree.dark_ranger
 actions+=/call_action_list,name=drcleave,strict=1,if=active_enemies>1&hero_tree.dark_ranger
 actions+=/call_action_list,name=sentcleave,strict=1,if=active_enemies>1&!hero_tree.dark_ranger
-actions+=/call_action_list,name=drst,strict=1,if=active_enemies=1&hero_tree.dark_ranger
-actions+=/call_action_list,name=sentst,if=active_enemies=1&!hero_tree.dark_ranger
+actions+=/call_action_list,name=drst,strict=1,if=hero_tree.dark_ranger
```

Comment:
As written, if a Dark Ranger had no recommendation after proceeding through the `drst` action list, or a recommendation from the `sentst` list ends up being ready sooner, we'll get the recommendation from the `sentst` list.  Is that desired?

If not, we would either want this to be a `run_action_list` entry or we would want the next entry to be `strict=1,if=!hero_tree.dark_ranger` so that these lists remain mutually exclusive.